client/src/components/makeForm/MakeForm.tsx

```diff
@@ -20,6 +20,12 @@ function MakeForm() {
       })
       .catch((error) => {
         toast.dismiss(loadingToast);
+
+        const { response } = error;
+        if (response.status === 401) {
```

Comment:
Nit: I think we should add the following to the imports: `import {StatusCodes} from "http-status-codes";` 
```suggestion
        if (response.status === StatusCodes.UNAUTHORIZED) {
```

Reply:
done